setup.py

```diff
@@ -41,7 +41,9 @@
           "decorator>=3.2.0",
           "appdirs>=1.4.0",
           "numpy>=1.6.0",
-          "dataclasses>=0.7;python_version<='3.6'"
+          "dataclasses>=0.7;python_version<='3.6'",
+          "Mako",
+          "lark-parser",
```

Comment:
I'd like to avoid the dependency on Mako, and I'd like the lark dependency to be an "extra" (e.g. `"tag_parsing"?).

Reply:
Done. For installing pytools with tag parsing, now one would do: `pip install pytools[tag_parsing]`.